src/eig.jl

```diff
@@ -1,6 +1,7 @@
-using Base.LinAlg: Eigen
+using Compat.LinearAlgebra: Eigen
+import Compat.LinearAlgebra: eigfact
 
-function Base.eigfact{T}(::Type{Positive{T}}, A::AbstractMatrix{T}, args...; tol=default_tol(A))
+function eigfact(::Type{Positive{T}}, A::AbstractMatrix{T}, args...; tol=default_tol(A)) where {T}
```

Comment:
`eigfact` is now deprecated.  Should use `eigen(A; permute=permute, sacel=scale)` instead.

Reply:
Made that change. With the change in API from eigfact to eigen, I decided to also drop 0.6.